modules/supporting_encapsulation.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/optimizing-cpu-usage.adoc
+
+[id="supporting-encapsulation_{context}"]
+= Supporting encapsulation
```

Comment:
The purpose of this section is unclear
Suggest this is clarified or section skipped

Reply:
FYI @djuran Tony is unexpectedly out sick, I will take over this PR and complete it in https://github.com/openshift/openshift-docs/pull/53033